lib/v2/oreno3d/radar.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    'oreno3d.com': {
+        _name: 'oreno3d',
+        ecchi: [
```

Comment:
`ecchi.oreno3d.com` is 404 for me.
So I guess it should be
```suggestion
        '.': [
```